stdlib/test/collections/test_list.mojo

```diff
@@ -296,6 +296,71 @@ def test_list_reverse_move_count():
     # Keep vec alive until after we've done the last `vec.data + N` read.
     _ = vec^
 
+def test_list_insert():
+    #
+    # Test the list [1, 2, 3] created with insert
+    #
+
+    vec = List[Int]()
+    vec.insert(len(vec), 1)
+    vec.insert(len(vec), 3)
+    vec.insert(1, 2)
+
+    assert_equal(len(vec), 3)
+    assert_equal(vec[0], 1)
+    assert_equal(vec[1], 2)
+    assert_equal(vec[2], 3)
+
+    vec.clear()
```

Comment:
**Suggestion** Instead of manually running `clear()` between the different parts of this function, it would read better to just use different variables, such as `v1 = List[Int]()`, `v2 = List[Int]()`, and so on.  What do you think?

Reply:
Yeah seems like a good idea